### PR TITLE
Ht17 encapsulation in fractions

### PR DESCRIPTION
**Pull request by mykolamuntian**
Add encapsulation in homework with a class of fractions . Hide fields, creat public get and set methods. Add Constructor(s)

### CODE REVIEW

File: src/com/muntian/ht17_Encapsulation/ActionsWithFractions.java [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+            secondSummand.setIntegerPart(makePositiveNumber(secondSummand.getIntegerPart()));
+            return subtraction(firstSummand, secondSummand);
+        }else if (firstSummand.getIntegerPart() < 0 && secondSummand.getIntegerPart() >= 0) {
+            firstSummand.setIntegerPart(makePositiveNumber(firstSummand.getIntegerPart()));
+            return subtraction(secondSummand, firstSummand);
+        }
+        return result;
+    }
+
+    private static void additionOfFractionalParts(Fractions firstSummand, Fractions secondSummand, Fractions result) {
```
Comment (strazhkoe) [inline src/com/muntian/ht17_Encapsulation/ActionsWithFractions.java]: Все арифметические операции необходимо перенести в класс Fraction и сделать их нестатическими

**Review by strazhkoe — CHANGES_REQUESTED** [review]

File: src/com/muntian/ht17_Encapsulation/ActionsWithFractions.java (hunk 1 above)
Comment (mykolamuntian) [reply src/com/muntian/ht17_Encapsulation/ActionsWithFractions.java]: Task was fixed according to the comments

File: src/com/muntian/ht20_Passport/Passport.java [hunk 2]
```diff
@@ -0,0 +1,103 @@
+package com.muntian.ht20_Passport;
+
+public class Passport {
```
Comment (strazhkoe) [inline src/com/muntian/ht20_Passport/Passport.java]: How did this class get here?

File: src/com/muntian/ht20_Passport/Visa.java [hunk 3]
```diff
@@ -0,0 +1,41 @@
+package com.muntian.ht20_Passport;
+
+public class Visa {
```
Comment (strazhkoe) [inline src/com/muntian/ht20_Passport/Visa.java]: How did this class get here?

**Review by strazhkoe — CHANGES_REQUESTED** [review]

File: src/com/muntian/ht20_Passport/Visa.java (hunk 3 above)
Comment (mykolamuntian) [reply src/com/muntian/ht20_Passport/Visa.java]: My mistake.  Already fixed.

**Review by strazhkoe — APPROVED** [review]

**Pull request merged**